pkg/oc/lib/newapp/app/pipeline.go

```diff
@@ -194,6 +194,14 @@ func (p *Pipeline) Objects(accept, objectAccept Acceptor) (Objects, error) {
 		}
 		if objectAccept.Accept(repo) {
 			objects = append(objects, repo)
+		} else if len(p.InputImage.Reference.Namespace) == 0 {
```

Comment:
this seems weird.  can you add a comment explaining it?

and does it breakdown in scenarios where the input and output imagestreamtags are in the same namespace?  (and the imagestream may or may not already exist)